### ISSUE TITLE
Rails 4.1.6: Undefined method app

### ISSUE THREAD

**Issue by nbibler** [envylabs/rapporteur#15]
Reported by @sshaw in #13:

```
/Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rapporteur-3.5.0/config/routes.rb:5:in `block in <top (required)>': undefined method `app' for #<Sprockets::Environment:0x007fd6f9c18ef0> (NoMethodError)
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/actionpack-4.1.16/lib/action_dispatch/journey/routes.rb:32:in `each'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/actionpack-4.1.16/lib/action_dispatch/journey/routes.rb:32:in `each'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rapporteur-3.5.0/config/routes.rb:5:in `any?'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rapporteur-3.5.0/config/routes.rb:5:in `<top (required)>'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/activesupport-4.1.16/lib/active_support/dependencies.rb:241:in `load'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/activesupport-4.1.16/lib/active_support/dependencies.rb:241:in `block in load'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/activesupport-4.1.16/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/activesupport-4.1.16/lib/active_support/dependencies.rb:241:in `load'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:40:in `block in load_paths'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:40:in `each'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:40:in `load_paths'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:16:in `reload!'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:26:in `block in updater'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/activesupport-4.1.16/lib/active_support/file_update_checker.rb:75:in `execute'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:27:in `updater'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/routes_reloader.rb:7:in `execute_if_updated'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application/finisher.rb:71:in `block in <module:Finisher>'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/initializable.rb:30:in `run'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:228:in `block in tsort_each'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:347:in `each'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:347:in `call'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:347:in `each_strongly_connected_component'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:226:in `tsort_each'
	from /Users/skyes/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/tsort.rb:205:in `tsort_each'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/application.rb:300:in `initialize!'
	from /Users/skyes/code/ruby/rails41-test/config/environment.rb:5:in `<top (required)>'
	from /Users/skyes/code/ruby/rails41-test/config.ru:3:in `require'
	from /Users/skyes/code/ruby/rails41-test/config.ru:3:in `block in <main>'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/builder.rb:55:in `instance_eval'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/builder.rb:55:in `initialize'
	from /Users/skyes/code/ruby/rails41-test/config.ru:in `new'
	from /Users/skyes/code/ruby/rails41-test/config.ru:in `<main>'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/builder.rb:49:in `eval'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/builder.rb:49:in `new_from_string'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/builder.rb:40:in `parse_file'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/server.rb:277:in `build_app_and_options_from_config'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/server.rb:199:in `app'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/server.rb:50:in `app'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/rack-1.5.5/lib/rack/server.rb:314:in `wrapped_app'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/server.rb:130:in `log_to_stdout'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/server.rb:67:in `start'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/commands_tasks.rb:81:in `block in server'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/commands_tasks.rb:76:in `tap'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/commands_tasks.rb:76:in `server'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/skyes/.rvm/gems/ruby-2.3.0@rails41/gems/railties-4.1.16/lib/rails/commands.rb:17:in `<top (required)>'
	from ./bin/rails:4:in `require'
	from ./bin/rails:4:in `<main>'
```

**Comment by nbibler** [comment]
@sshaw: I found at least _an_ issue in Rails 4.0 and 4.1. I've pushed a fix. I think it should correct your reported issue as well. If you could, please pull `master` from your Gemfile (as of 5462dbe) and let me know if that corrects your issue.

**Comment by sshaw** [comment]
Fixes it. Thanks.

**Comment by nbibler** [comment]
Published as v3.5.1. Update your Gemfile and you should be good!
